main.py

```diff
@@ -0,0 +1,15 @@
+from flask import Flask
+from flask_restful import Resource, Api
+from data.calendar_json import calendars
+
+app = Flask(__name__)
+api = Api(app)
+
+class Calendars(Resource):
+    def get(self):
+        return calendars
+
+api.add_resource(Calendars, '/calendars')
+
+if __name__ == '__main__':
+    app.run(debug=False)
```

Comment:
[How to run a Flask Application](https://www.twilio.com/blog/how-run-flask-application)

Reply:
Later on we should play with some of these flags for `app.run`

`use_debugger` This flag will let us use the built-in Werkzeug debugger, which is the `pry` we have been looking for. [Documentation](https://werkzeug.palletsprojects.com/en/1.0.x/debug/)

Reply:
We have to make sure `debug=False` when pushing up.
But when we are developing locally, this has to change to `debug=True`.

TODO: Figure out how to automate this. Maybe by using environment variables?

Reply:
It would be cool to write a script for Travis to run to edit this in its build. [snippet](https://www.kite.com/python/answers/how-to-edit-a-specific-line-in-a-text-file-in-python#:~:text=Use%20file.,at%20a%20certain%20line%20number.)